docs/topics/modules/interpreting_stack_report.adoc

```diff
@@ -0,0 +1,47 @@
+[#interpreting_stack_report]
+= Interpreting your Stack Report
+
+The Stack Report comprises three components:
+
+== Stack report: An overall stack analysis
+The Stack report scans the total number of components in your stack, and reports those analyzed by {osio} and those unknown to it.
+
+It displays:
+
+Security Alerts:: {osio} examines the Common vulnerabilities and exposures (CVEs) for the components of the stack and highlights vulnerabilities, if any.
+Usage:: It analyzes your overall stack composition, compares it to other similar stacks and displays the number of components in your stack that are not commonly used in other similar stacks or that rarely go together.
+Licenses:: It analyzes the licenses used in your stack, and displays the number of unknown licenses. It also displays the number of license conflicts at the stack level and the conflicting licenses used in the stack on mouse hover.
+
+== Detail analysis of your stack components
+The detailed component analysis lists all the components in the stack, analyzes them on the basis of various parameters, and recommends relevant Alternate Components.
+
+It helps you assess the credibility of your stack components by displaying component level details such as:
+
+Versions:: It lists Current, Recommended, and the Latest available version of the components in your stack so that you can select a suitable version for your stack.
+
+Security Issue:: Based on the CVEs of the stack components, {osio} identifies security vulnerabilities in your stack components. On mouse hover, it displays the CVE scores of components identified with security vulnerabilities.
+
+License:: It lists Licenses for each component, flags Unknown and Conflicting licenses, and recommends *Alternate Components* in case of *License Outliers*.
+
+OSIO Usage:: It tracks the number of projects in {osio} that use the same component. It recommends *Alternate Components* in case of *Usage Outliers*.
+
+GitHub Statistics:: {osio} tracks GitHub statistics such as Usage and number of Watchers for the components of your stack.
```

Comment:
Usage and Watchers should be marked-up.

statistics such as -> statistics, such as